addons/survey/tests/test_survey_invite.py

```diff
@@ -75,7 +75,7 @@ def test_survey_invite(self):
         invite_form.deadline = fields.Datetime.to_string(deadline)
 
         invite = invite_form.save()
-        invite.action_invite()
+        invite.with_context(active_model="survey.survey").action_invite()
```

Comment:
Would be nice to avoid context management in the override of "_notify_get_recipients_groups", should find a way to modify the button title based on other means (model fields ideally).